services/user_service/test_user_service.py

```diff
@@ -0,0 +1,81 @@
+
+from database_connector import DatabaseAdmin
+import mocker
+
+
+databaseURL = "https://lohyna-user-service-default-rtdb.firebaseio.com/"
+credentials_path = "firebase_test.json"
+database_manager = DatabaseAdmin(databaseURL, credentials_path)
```

Comment:
Can you replace the firebase connection with a mock class, so you don't need any remote calls for tests?

Also, you can replace all additional setup with [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html).

Also, try running tests with `-rA` flag, so you won't need additional logging of test names, probably.

Reply:
@arihilmir no, you can not replace it with fixtures. Initially, tests have been written with fixtures, but as I figured out, firebase will throw an exception while using fixtures. The case here, that you can have just one connection to the firebase, when fixture has been called before each tests execution.

Reply:
Well, we can mock the firebase, or provide some global env setup and tear down.

Or use something dependency injection for firebase.

Reply:
there are no info how to mock firebase in python. Only in js there is mock library. I do not think that spending more time here is reasonable.